includes/wikia/parser/templatetypes/handlers/ReferencesTemplate.class.php

```diff
@@ -2,11 +2,25 @@
 
 class ReferencesTemplate {
 	/**
-	 * @desc return simple <references /> parser tag
-	 *
+	 * @desc return simple <references /> parser tag if the original text contains it
+	 * @param $text string
 	 * @return string
 	 */
-	public static function handle() {
-		return '<references />';
+	public static function handle( $text ) {
+		$error_setting = libxml_use_internal_errors( true );
```

Comment:
To avoid flood of PHP Warnings that are caused by lack of compatibility with valid HTML5 tags, i.e. figure. If you think it makes sense, we could catch them before libxml_clear_errors() statement and re-log with lower priority; based on currently logged libxml warnings it does not seem to make sense to me.